tests/integration_tests/data_steward/cdr_cleaner/cleaning_rules/remove_non_matching_participant_test.py

```diff
@@ -47,22 +46,76 @@
         INSERT INTO `{{fq_table_name}}` 
         (observation_id, person_id, observation_concept_id, observation_date, observation_type_concept_id)
         VALUES
```

Comment:
@hiro-mishima I spent some time yesterday looking at this integration test.  It is duplicating a lot of work that the base class does for free.  I'm adding a patch file for your review with this test.  We can talk about it whenever you would like.
[remove_non_matching_participant_test_patch.txt](https://github.com/all-of-us/curation/files/9376283/remove_non_matching_participant_test_patch.txt)

Let me know if you need help applying the patch file.

Reply:
Thanks for the patch file! The patch makes sense to me. Now the test looks cleaner 🎉

Reply:
This addition to the `data_steward/gcloud/bq/__init__.py` will be quite helpful. Thanks.